redis/exceptions.py

```diff
@@ -57,6 +57,10 @@ class NoPermissionError(ResponseError):
     pass
 
 
+class ModuleError(RedisError):
```

Comment:
`ModuleError` should inherit from `ResponseError` rather than `RedisError`.

Reply:
OK